os/pm/pm_initialize.c

```diff
@@ -113,11 +113,6 @@ void pm_initialize(void)
 	g_pmglobals.stime = clock_systimer();
 	g_pmglobals.btime = clock_systimer();
 
-#ifdef CONFIG_PM_TICKSUPPRESS
-	/* Registers a handler to be called when the core wakes up */
```

Comment:
Please separate pm_initialize.c and arch.h changes into different commit

Reply:
done~